crates/provider/src/shared/constants.rs

```diff
@@ -0,0 +1,6 @@
+// CONSTANTS
+pub const DEFAULT_REMOTE_DIR: &str = "/cfg";
+pub const DEFAULT_DATA_DIR: &str = "/data";
+pub const LOCALHOST: &str = "127.0.0.1";
```

Comment:
Maybe we should use the IpAddr from the stdlib, this is what I use in the configuration crate